Sources/CLI/RunCommand.swift

```diff
@@ -294,6 +302,39 @@ struct ProcessIO {
         )
     }
 
+    static func streamStdin(
+        from: OSFile,
+        to: OSFile,
+        buffer: UnsafeMutableBufferPointer<UInt8>,
+        onErrorOrEOF: () -> Void,
+    ) {
+        while true {
+            let r = from.read(buffer)
+            if r.read > 0 {
+                let view = UnsafeMutableBufferPointer(
+                    start: buffer.baseAddress,
+                    count: r.read
+                )
+
+                let w = to.write(view)
+                if w.wrote != r.read {
```

Comment:
destructure into `(bytesWritten, _)`

make it obvious that action is not used (and prompts the reviewer to check that an `w.action is not ok ⇔ bytesRead != bytesWritten`)